core/src/main/java/org/infinispan/statetransfer/InboundTransferTask.java

```diff
@@ -118,7 +118,7 @@ public Address getSource() {
             return completionFuture;
          }
          if (trace) {
-            log.tracef("Requesting segments %s of cache %s from node %s", segmentsCopy, cacheName, source);
+            log.tracef("Requesting state from node %s for segments %s", source, segmentsCopy);
```

Comment:
I think the cache name was already in the NDC. Also, none of the other messages contain the cache name.